src/Faker/Provider/ar_MA/Internet.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Faker\Provider\ar_MA;
+
+class Internet extends \Faker\Provider\Internet
+{
+    protected static $userNameFormats = array(
+        '{{lastNameAscii}}.{{firstNameAscii}}',
+        '{{firstNameAscii}}.{{lastNameAscii}}',
+        '{{firstNameAscii}}##',
+        '?{{lastNameAscii}}',
+    );
+    protected static $safeEmailTld = array(
+        'com', 'jo', 'me', 'net', 'org',
+    );
+
+    protected static $tld = array(
+        'biz', 'com', 'info', 'jo', 'net', 'org',
+    );
+
+    protected static $lastNameAscii = array(
```

Comment:
please add a datasource and add a line break for every capital letter. Same for first name.

Reply:
ok